docs/source/reference/tpu.rst

```diff
@@ -54,10 +132,10 @@ SkyPilot automates the above process with a simple interface:
          runtime_version: 2.5.0 # TPU software version to be used.
 
 The above YAML considers :code:`n1-highmem-8` as the host machine and :code:`tpu-v2-8` as the TPU node resource.
-You may modify the host instance type or TPU type as you wish.
-To show more TPU accelerators, you may run the command :code:`sky show-gpus`.
+You can modify the host instance type or the TPU type.
+
+Here is a complete task YAML that runs `MNIST training <https://cloud.google.com/tpu/docs/run-calculation-jax#running_jax_code_on_a_tpu_vm>`_ on a TPU Node using TensorFlow.
 
-Now, we show a complete YAML for running `MNIST training <https://cloud.google.com/tpu/docs/tutorials/mnist-2.x>`_ on TPU node with TensorFlow.
 
 .. code-block:: yaml
 
```

Comment:
After L150, add: `# We use SkyPilot Storage to mount a GCS bucket to /dataset.`

L148 and L132: `<two spaces> # optional, TPU runtime version`

L142, L157, L170: remove? To be consistent with above.